Header header
float32[6] swing_phase
float32[6] stance_phase
float32[6] touch_possibility
